### ISSUE TITLE
README to setup the thikira backend.

### ISSUE THREAD

**Issue by ashishkharcheiu** [thikira/thikira-back-end#2]
Could you please add steps to setup the backend? I want to run the backend on my pc.

Also, did you use mongodb or mysql as database? I am new to NodeJS.

**Comment by jepangLee** [comment]
Thank you for your attention to my repository
I will post it as soon as possible.

**Comment by jepangLee** [comment]
i use both database

**Comment by ashishkharcheiu** [comment]
Thankyou! @jepangLee Waiting for readme! I do want to test you android app

**Comment by jepangLee** [comment]
finish!

**Comment by ashishkharcheiu** [comment]
Thank you! @jepangLee! I will go through it.
Your work looks interesting, so please keep on updating README as much as possible. It helps a lot!

**Comment by kimdohun0104** [comment]
Great to see you! 

> Thankyou! @jepangLee Waiting for readme! I do want to test you android app

I just started to wonder 'want to test you android app' suppose to mean? 
You mean you want to test server with [my work](https://github.com/thikira/ThikiraAndroid)?
If you do, don't mind asking me.

I just ask you because, you seemed to interested in Android development too.
I always prepared for help!

Thank you!

**Comment by ashishkharcheiu** [comment]
@kimdohun0104 Yes, basically I want to TEST means RUN the app on my device and then check the source code. Thanks for open-source.
